### ISSUE TITLE
Remove Channels from AE2 so its like AE1 from 1.6.4 with stuff from AE2

### ISSUE THREAD

**Issue by mariodk** [The-1-7-10-Pack/The-1.7.10-Pack#149]
i kinda dislike the Channels system from AE2
its kinda set a limit on machine setups like my little Barrel storage room i had in 1.6.4
where i put storage busses on each barrel aswell as item pipes but now im limited to only 8-32 instead of a full room 3 block high and about as big as a chunk
but with AE2 and with basically every modpack(with servers) i can find all has these Channels turned on in config

so basically AE1 in 1.7.10 but with multi busses/stuff on 1 block,new auto crafting CPUs/stuff

also im sure others also kinda dislike that channel system but yet so many modpacks add them

**Comment by Flukethoughts** [comment]
never going to happen
